GNOME 48: Cannot open/switch apps using the launcher

After updating to GNOME 48, alongside updating both `pop-launcher` and `pop-shell` to their latest git versions, I don't seem to be able to open/switch apps using the launcher. When enabling logs for `pop-shell` and checking `journalctl`, here is what I get when I attempt to hit Enter to open/switch apps:
```
Mar 27 10:50:47 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:47 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:47 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [ERROR] failed to create SearchOption: TypeError: this.scroller.get_vscroll_bar is not a function
Mar 27 10:50:48 adachi gnome-shell[269225]: pop-shell: [INFO] stopping pop-launcher services
```

--- a/src/search.ts
+++ b/src/search.ts
@@ -373,12 +373,7 @@ export class Search {
 
         this.list.show();
 
-        const vscroll = (this.scroller as any).get_vscroll_bar();
-        if ((this.scroller as any).vscrollbar_visible) {
-            vscroll.show();
-        } else {
-            vscroll.hide();
-        }
+        (this.scroller as any).set_policy(St.PolicyType.NEVER, St.PolicyType.AUTOMATIC);
 
         if (id === 0) {
             GLib.idle_add(GLib.PRIORITY_DEFAULT, () => {

